aptos-move/block-executor/src/executor.rs

```diff
@@ -511,10 +550,16 @@ where
         let mut block_limit_processor = shared_commit_state.acquire();
 
         while let Some((txn_idx, incarnation)) = scheduler.try_commit() {
+            let mut executed_at_commit = false;
             if !Self::validate_commit_ready(txn_idx, versioned_cache, last_input_output)? {
```

Comment:
we should rename "validate_commit_ready" to "validate_delayed_fields" or something more obvious, or at least add a comment doc for the function. "commit_ready" is too general for readers to understand what's happening